04.bowling/bowling.rb

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+score = ARGV[0]
+scores = score.split(',')
+
+shots = scores.map do |s|
+  if s == 'X'
+    shots = [10, 0]
+  else
+    s.to_i
+  end
+end
+
+shots = shots.flatten
```

Comment:
ちょっと上級テクニックですが、こんなふうに書けるかも？

```suggestion
shots = scores.flat_map { |s| s == 'X' ? shots = [10, 0] : s.to_i }
```

Reply:
現在チェリー本２版を読み進めており、ちょうど4.3.5 で`do ... end` でなく`{}`で書く方法について学んでいました。
`{}`のブロックをお手本とさせていただきます。🙏

ありがとうございました！